core/drivers/crypto/hisilicon/include/hpre_main.h

```diff
@@ -0,0 +1,101 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023 HiSilicon Limited.
+ */
+#ifndef _HPRE_MAIN_H
+#define _HPRE_MAIN_H
+
+#include <rng_support.h>
```

Comment:
this header file does not need _rng_support.h_

Reply:
Some asymmetric algorithms will be added later, and they will use the interface of `hw_get_random_bytes()` to get random data as private key. So the header file rng_support.h is included.

Reply:
You should add this inclusion on the commit that need it.